nixos/tests/kexec.nix

```diff
@@ -18,28 +16,32 @@ import ./make-test-python.nix ({ pkgs, lib, ... }: {
 
     node2 = { modulesPath, ... }: {
       virtualisation.vlans = [ ];
+      environment.systemPackages = [ pkgs.hello ];
       imports = [
         "${modulesPath}/installer/kexec/kexec-boot.nix"
+        "${modulesPath}/profiles/minimal.nix
```

Comment:
No reason other than performance I suppose?

Reply:
yes, building the squashfs takes forever, so I wanted to minimize that a bit while testing and figured other people wanted the same